susemanager/bin/server-migrator.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+spacewalk-service stop
```

Comment:
I see we stop the services, but we don't start them at the end of the script.

This could be fine, as I guess we want the user to review what the script did, specially if there were errors. But I wonder if we should not warn the user about and ass a warning.

Something like:
```
echo "WARNING: This script will stop all Uyuni services and perform the migration. After it is complete, please review the output and if there were no errors then run 'spacewlak-service start`"
read -n 1 -s -r -p "Press any key to start the migration or CTRL+C to cancel...";
```
AFAIK we don't call this script automatically from anywhere else, so we should be fine asking the user for confirmation.

Reply:
Simply starting services after migration cannot work. Machine needs reboot since the whole base OS has been replaced. I will add some echo...

Reply:
Right!

Then adding the `echo` at the end of the script should be fine.